packages/visx-react-spring/src/axis/AnimatedTicks.tsx

```diff
@@ -20,63 +20,56 @@ export default function AnimatedTicks<Scale extends AxisScale>({
   ticks,
   animationTrajectory,
 }: TicksRendererProps<Scale> & { animationTrajectory?: AnimationTrajectory }) {
-  const animatedTicks = useTransition(
-    ticks,
-    tick => `${tick.value}`,
-    useLineTransitionConfig({ scale, animateXOrY: horizontal ? 'x' : 'y', animationTrajectory }),
-  );
+  const animatedTicks = useTransition(ticks, {
+    ...useLineTransitionConfig({
+      scale,
+      animateXOrY: horizontal ? 'x' : 'y',
+      animationTrajectory,
+    }),
+    keys: (tick: { value: any }) => `${tick.value}`,
```

Comment:
wonder if we could do something like this instead of using `any`

```tsx
// add this to the import above
import { ComputedTick, ... } from '@visx/axis/lib/types';

// in hook
keys: (tick: ComputedTick<Scale>) => `${tick.value}`,
```

Reply:
Great point! I'll change it.